### DISCUSSION TITLE
Detours releases? Is "4.0.1" version number meaningless?

### DISCUSSION THREAD

**Discussion by DrusTheAxe** [microsoft/Detours#246] [Q&A] [answered]
Does Detours make releases? I see commits but nothing about any sort of official or formalized release. Are users supposed to grab the latest repo source and build for themselves? And lather/rinse/repeat when there's updates?

I've got a copy of Detours source v4.0.1 that's at least a year old. I see commits since but no version change. How am I supposed to tell there's changes and what are they called? Ignore the version as useless and track the timestamp when I made my copy?

Please advise

**Comment by sylveon** [comment, marked as answer]
https://github.com/microsoft/Detours/releases/tag/v4.0.1 ?

**Reply by DrusTheAxe** [reply]
Ah! Didn't notice the 'Releases' link on the right of the main page. Sheepish grin

Now the obvious question: [dtarditi](https://github.com/dtarditi) released this Apr 16, 2018. No release in the past 3+ years? GitHub lists 3 pages of commits and while many (most?) seem to be docs, comments or build-time I spot some enhancements and several fixes e.g. [Fix ARM64 detour_skip_jmp.](https://github.com/microsoft/Detours/pull/8)

Does Detours have a release plan or cadence or...? What should my expectations be? Doesn't seem to be a dead project given the regular commit activity but doesn't seem to have any release plan or schedule or process other than "clone the repo when you want updates and work by timestamp as '4.0.1' isn't meaningful". Am I missing something?

**Reply by sylveon** [reply]
It seems like there was a plan for a new release but it died: https://github.com/microsoft/Detours/issues/163#issuecomment-790981295

Maybe @dtarditi or @bgianfo knows what happened

**Reply by bgianfo** [reply]
If I'm being honest life just happened. There are still some PR's I want to make traction on and get merged before we snap the next release. But maybe it's not worth waiting on them. 

I will try to see if it makes sense to do some testing and release what we have at the moment.

Apologies for the delay.

**Reply by DrusTheAxe** [reply]
ping

Any news?

**Reply by DrusTheAxe** [reply]
ping ?
